dashboard/test/models/authentication_option_test.rb

```diff
@@ -65,7 +76,7 @@ class AuthenticationOptionTest < ActiveSupport::TestCase
   test 'user can have multiple authentication options' do
     assert_creates(User) do
       user = create(:user, :with_google_authentication_option, :with_clever_authentication_option)
-      assert_equal 3, user.authentication_options.length
+      assert_equal 3, user.authentication_options.count
```

Comment:
Not sure why this wasn't failing before, this should be `count` which will actually query the number of `authentication_options` associated with a user, whereas `length` works on a value that has already been returned from a query (ie. `user.authentication_options.all.length`).